### ISSUE TITLE
Changes for 01 Styles / 01 Custom CSS

### ISSUE THREAD

**Issue by crsanti** [Lemoncode/webpack-1.x-by-sample#33]
Hi, i found some changes we could do:

> - Configure webpackconfig.

We could change this to _Configure webpack.config.js_.

> Prerequisites, you will need to have nodejs installed in your computer. If you want to follow this guide you will need to take as starting point sample "00 Intro/03 Jquery"

End sentene with a full stop and change sample to _00 Intro / 04 Jquery_.

``` css
.redbkg {
  background-color:  red
}
```

Add trailing semicolon, also in _students.js_. :P

``` javascript
import {getAvg} from "./averageService";

$('body').css('background-color', 'blue');

const scores = [90, 75, 60, 99, 94, 30];
...
```

In the previous _students.js_ inside sample _00 Intro / 04 Jquery_ we have this code.
However after we add _redbkg_ class the figure shows this:

> ![Demo01_00_CSS.png](https://raw.githubusercontent.com/Lemoncode/webpack-1.x-by-sample/master/99%20Readme%20Resources/02%20Webpack/Demo01_00_CSS.png)

There's no blue background because the jQuery line was removed. This should be notified.

> **In next demos we will learn how to ask webpack to separate the css into separate files**,
>  depending on your scenario you could choose one way or the other.

How does this sound?

> **In next demos we will learn how to ask webpack to separate our styles from bundle.js into separate css output file**, depending on your scenario you could choose one way or the other.

**Comment by linuxonrails** [comment]
I think, this issue should be closed :-/

**Comment by crsanti** [comment]
Closed in favor of #54
